src/app/components/Article.jsx

```diff
@@ -10,7 +11,36 @@ const Headline = styled.h1`
   font-size: 2em;
 `;
 
+const renderModel = ({ blocks }) =>
+  blocks.map(block => {
+    const { type, blockId } = block;
+    const blockString = JSON.stringify(block);
+
+    let Element = 'p';
```

Comment:
This is fine at the moment. Will be interesting to see how it develops as it gets significantly more convoluted.